packages/babel/src/traversal/scope/index.js

```diff
@@ -540,7 +540,12 @@ export default class Scope {
       }
     } else if (path.isClassDeclaration()) {
       this.registerBinding("let", path);
-    } else if (path.isImportDeclaration() || path.isExportDeclaration()) {
+    } else if (path.isImportDeclaration()) {
+      var specifiers = path.get("specifiers");
+      for (var specifier of (specifiers: Array)) {
+        this.registerBinding("module", specifier);
+      }
+    } else if (path.isExportDeclaration()) {
```

Comment:
Would be good to get the same specifier behaviour for export declarations too. Changing the condition to `path.isModuleDeclaration()` and adding `this.registerBinding("module", this.get("declaration"))` after the specifier loop should be enough.